app/views/recipes/show.html.haml

```diff
@@ -0,0 +1,24 @@
+.page-header
+  %h1.text-uppercase.text-center Recipes Details
+
+.row
+  .col-md-4.col-md-offset-4
+    %p
+      %strong Name: 
+      #{@recipe.name}  
+    %p
+      %strong Preparation:
+      %br
+      #{@recipe.preparation}
+    %p
+      %strong Created On: 
+      #{@recipe.created_at.strftime('%d %B %Y')}
```

Comment:
Also, use this approach to format dates:
http://stackoverflow.com/questions/3879680/how-can-i-make-rails-3-localize-my-date-formats
You will have to add the format to the `en.yml` locale file.